packages/menu/menu-group.md

```diff
@@ -76,3 +121,112 @@ An `<sp-menu-group>` can be used to organize `<sp-menu-item>`s in an `<sp-memu>`
+</p>
+<sp-popover open style="position: relative">
+    <sp-menu
+        label="What are your favorite parks?"
+        style="width: 200px"
+        onchange="this.parentElement
+                    .previousElementSibling
+                    .querySelector(`#${arguments[0].target.id}-value`)
+                    .textContent = arguments[0].target.value">
+        <sp-menu-group
+            id="multiple-group-1"
+            selects="multiple"
+        >
+            <span slot="header">New York</span>
+            <sp-menu-item>
+                Central Park
+            </sp-menu-item>
+            <sp-menu-item>
+                Flushing Meadows Corona Park
+            </sp-menu-item>
+            <sp-menu-item>
+                Prospect Park
+            </sp-menu-item>
+        </sp-menu-group>
+        <sp-menu-group
+            id="multiple-group-2"
+            selects="multiple"
+        >
+            <span slot="header">San Francisco</span>
+            <sp-menu-item>
+                Golden Gate Park
+            </sp-menu-item>
+            <sp-menu-item>
+                John McLaren Park
+            </sp-menu-item>
+            <sp-menu-item>
+                Lake Merced Park
+            </sp-menu-item>
+        </sp-menu-group>
+    </sp-menu>
+</sp-popover>
+```
+
+</sp-tab-panel>
+<sp-tab value="selects-inherit">Inherit</sp-tab>
+<sp-tab-panel value="selects-inherit">
+
+<!-- prettier-ignore -->
+```html demo
+<p>
+    Your favorite park is: <span id="inherit-group-value"></span>
+</p>
+<sp-popover open style="position: relative">
+    <sp-menu
+        id="inherit-group"
+        label="What are your favorite parks?"
+        style="width: 200px"
+        selects="single"
+        onchange="this.parentElement
+                    .previousElementSibling
+                    .querySelector(`#${arguments[0].target.id}-value`)
+                    .textContent = arguments[0].target.value">
+        <sp-menu-group
+            id="inherit-group-1"
+            selects="inherit"
+        >
+            <span slot="header">New York</span>
+            <sp-menu-item>
+                Central Park
+            </sp-menu-item>
+            <sp-menu-item>
+                Flushing Meadows Corona Park
+            </sp-menu-item>
+            <sp-menu-item>
+                Prospect Park
+            </sp-menu-item>
+        </sp-menu-group>
+        <sp-menu-group
+            id="inherit-group-2"
+            selects="inherit"
+        >
+            <span slot="header">San Francisco</span>
+            <sp-menu-item>
+                Golden Gate Park
+            </sp-menu-item>
+            <sp-menu-item>
+                John McLaren Park
+            </sp-menu-item>
+            <sp-menu-item>
+                Lake Merced Park
+            </sp-menu-item>
+        </sp-menu-group>
+    </sp-menu>
+</sp-popover>
+```
+
+</sp-tab-panel>
+</sp-tabs>
```

Comment:
Does this need an accessibility guidelines section or is it redundant as there isn't really anything to do?
I'm happy either way but wanted to make sure the omission was deliberate.

Reply:
I'm going to add it and cross reference the other components.